cdk/cdk-resources/encryption-at-rest/src/integ.default.ts

```diff
@@ -0,0 +1,22 @@
+import * as cdk from 'aws-cdk-lib';
+import { CfnEncryptionAtRest } from './index';
```

Comment:
could you update https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/intmdb-669-updatel1construct-encryptionatrest/cdk/cdk-resources/encryption-at-rest/test/index.test.ts to use the profile? Thanks!

Reply:
@andreaangiolillo  added Profile Prop